docs/data-sources/cluster.md

```diff
@@ -103,8 +103,13 @@ Required:
 
 - `requirements` (Attributes List) List of requirements for the node pool (see [below for nested schema](#nestedatt--features--karpenter--qovery_node_pools--requirements))
 
+Optional:
+
+- `default_override` (Attributes) Defines some overridden options for Qovery default node pool (see [below for nested schema](#nestedatt--features--karpenter--qovery_node_pools--default_override))
+- `stable_override` (Attributes) Defines some overridden options for Qovery stable node pool (see [below for nested schema](#nestedatt--features--karpenter--qovery_node_pools--stable_override))
+
 <a id="nestedatt--features--karpenter--qovery_node_pools--requirements"></a>
-### Nested Schema for `features.karpenter.qovery_node_pools.requirements`
+### Nested Schema for `features.karpenter.qovery_node_pools.stable_override`
```

Comment:
Here the documentation generated is wrong, it's a behavior that we already have on some other resources, i.e on applications (see screenshots below).
Working on fixing the doc in a separate PR
![image](https://github.com/user-attachments/assets/f3647501-7975-4a6a-834d-afb4a1107bd7)
![image](https://github.com/user-attachments/assets/7dc86dd4-f4e9-4626-afd2-2571eead5be6)